Libraries/LibC/crt0.cpp

```diff
@@ -38,6 +38,9 @@ extern void _init();
 extern char** environ;
 extern bool __environ_is_malloced;
 
+// Tell the compiler that this may be called from somewhere else.
+int _start(int argc, char** argv, char** env);
```

Comment:
The commit message seems disconnected from the code change here.

Reply:
I'm not sure what you mean? I agree that "LibC: Tell compiler about invisible call to _init" and "Tell the compiler that this may be called from somewhere else." isn't word-for-word identical, but both are about the same thing: Declaring a function, so that the compiler knows that calls into `_init` may happen from outside this compilation unit. Otherwise, if the function isn't declared (separately from its definition), the heuristic in `-Wmissing-declarations` thinks that no other place can call the function, and issues a warning that maybe the function should be `static`.

For special functions like `_start` or `__cxa_pure_virtual`, this is a bit silly, sure, but I can't change gcc to ignore those special functions .

Reply:
The commit message talks about `_init` and you're adding a prototype for `_start`

Reply:
Oh how silly XD

You're right, it's supposed to say `_start` everywhere.